comit/src/expiries.rs

```diff
@@ -326,9 +326,9 @@ impl<A, B> fmt::Display for Expiries<A, B> {
 
 // Its super easy to mix these up, add types so the compiler saves us.
 #[derive(Clone, Copy, Debug, PartialEq, Eq, PartialOrd, Ord)]
-pub struct AlphaOffset(Duration);
+pub struct AlphaOffset(pub Duration);
```

Comment:
Hmm, not sure if these should be public. What do you think @tcharding?

There exist `From` conversions, so one could argue that if I can convert any `Duration` into one of these, then they might as well be public.

Reply:
As a general rule I'd say that if you need conversion, then use the `From`/`Into` traits.
Conversion is not a good enough excuse to expose internals.

Reply:
@rishflab Can you use the conversion functions please instead of making these public?

Reply:
doen